jmeter-service/chart/templates/deployment.yaml

```diff
@@ -170,4 +167,5 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      terminationGracePeriodSeconds: 90
\ No newline at end of file
+      terminationGracePeriodSeconds: {{ .Values.jmeterservice.gracePeriod | default 120 }}
+      
```

Comment:
this newline still seems off...

Reply:
but there is no missing newline error anymore no? The problem before was, that the newline was with the intendation, now it's without it...